components/director/internal/repo/upsert.go

```diff
@@ -0,0 +1,61 @@
+package repo
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/lib/pq"
+
+	"github.com/kyma-incubator/compass/components/director/internal/persistence"
+	"github.com/pkg/errors"
+)
+
+type Upserter struct {
+	tableName          string
+	createColumns      []string
+	conflictingColumns []string
+	updateColumns      []string
+}
+
+func NewUpserter(tableName string, createColumns []string, conflictingColumns []string, updateColumns []string) *Upserter {
+	return &Upserter{
+		tableName:          tableName,
+		createColumns:      createColumns,
```

Comment:
Why not just`entityColumns` or `columns`?

Reply:
Just `columns` wouldn't tell much about them when there are 3 "columns" in arguments in total. Not sure about `entityColumns`, why do you think that name would fit better?